src/mixer/audio.ts

```diff
@@ -452,6 +503,30 @@ export class AudioEngine extends ((EngineEmitter as unknown) as {
     this.players[number] = undefined;
   }
 
+  public setPFL(number: number, enabled: boolean) {
+    var routeMainOut = true;
+    var player = this.getPlayer(number);
+
+    if (player) {
+      player.setPFL(enabled);
+    }
+
+    var i;
+    for (i = 0; i < this.players.length; i++) {
+      player = this.getPlayer(i);
+      if (player?.getPFL()) {
+        // PFL is enabled on this channel, so we're not routing the regular output to H/Ps.
+        routeMainOut = false;
+        console.log("Player", i, "is PFL'd.");
+      } else {
+        console.log("Player", i, "isn't PFL'd.");
+      }
+    }
+    console.log("Routing main out?", routeMainOut);
+
+    this._connectFinalCompressor(routeMainOut);
```

Comment:
Er, I don't quite fully understand how this works. The way I read it, when a single player is PFL'd, we don't route the final compressor to headphones - which is fair, but where do the other players get routed then (assuming only one is PFL'd)?